xwiki-platform-core/xwiki-platform-oldcore/src/main/java/com/xpn/xwiki/web/XWikiServletURLFactory.java

```diff
@@ -607,12 +608,60 @@ public URL createAttachmentURL(String filename, String spaces, String name, Stri
                         querystring, xwikidb, context);
                 }
             } catch (XWikiException e) {
-                if (LOGGER.isErrorEnabled()) {
-                    LOGGER.error("Exception while trying to get attachment version !", e);
+                LOGGER.error("Exception while trying to get attachment version !", e);
```

Comment:
extra space before the "!". ;) I also suggest adding more information in the log such as the attachment reference.

Reply:
Actually this is not from you... :)